src/plugins/workspace/server/saved_objects/saved_objects_wrapper_for_check_workspace_conflict.ts

```diff
@@ -49,6 +68,16 @@ export class WorkspaceConflictSavedObjectsClientWrapper {
       options: SavedObjectsCreateOptions = {}
     ) => {
       const { workspaces, id, overwrite } = options;
+
+      if (workspaces?.length && (this.isDataSourceType(type) || this.isConfigType(type))) {
+        // For 2.14, data source can only be created without workspace info
```

Comment:
is it more of `Starting 2.14, data source can not be created if there is workspace inside options` if I understand it correctly, or will the statement change in newer versions?

Reply:
The statement will change to `only dashboard admin can create data source` in 2.15.